controllers/useraccount/useraccount_controller.go

```diff
@@ -484,6 +500,93 @@ func (r *Reconciler) deleteUser(ctx context.Context, userAcc *toolchainv1alpha1.
 	return true, nil
 }
 
+// Returns `true` if the associated resources (configMap, role and role-binding) created for a user by console is deleted, `false` otherwise with the underlying error.
+// This function only looks for these resources in the namespace - openshift-console-user-settings -
+func (r *Reconciler) deleteUserResources(ctx context.Context, userID string) (bool, error) {
+
+	// new user will have a label introduced which can help retrieve resources, otherwise older ones will have to rely on the name of the resource being of type `user-settings-<UID>`
+	// delete ConfigMap, Role and RoleBinding
+	if deleted, err := r.deleteConfigMap(ctx, userID); err != nil || deleted {
+		return deleted, err
+	}
+	if deleted, err := r.deleteRole(ctx, userID); err != nil || deleted {
+		return deleted, err
+	}
+	if deleted, err := r.deleteRoleBinding(ctx, userID); err != nil || deleted {
+		return deleted, err
+	}
+	return false, nil
+}
+
+func (r *Reconciler) deleteConfigMap(ctx context.Context, uid string) (bool, error) {
```

Comment:
```suggestion
func (r *Reconciler) deleteConfigMap(ctx context.Context, userUID string) (bool, error) {
```
Just in case it's not clear. `UID` means unique identifier. Not user ID :) Every kube resource has an UID. So we need to make it clear that it's the corresponding user's UID. Not the Config Map UID.

Reply:
done in [de45e6b](https://github.com/codeready-toolchain/member-operator/pull/568/commits/de45e6b31fcd13fe55a985f7614164dfb008a934)